specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementGetPrivateLinkGroupResource.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "privateLinkSubResourceName": "privateLinkSubResourceName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/privateLinkResources/Gateway",
```

Comment:
>Gateway [](http://example.com/codeflow?start=131&length=7)

It should be the name passed in the request parameter privateLinkSubResourceName